lua/nvim-peekup/config.lua

```diff
@@ -1,5 +1,8 @@
 local reg_chars = {}
-local chars = 'abcdefghijklmnopqrstuvwxyz0123456789*+-%'
+local chars = 'abcdefghijklmnopqrstuvwxyz0123456789-%'
+if vim.fn.has("clipboard") == "1" then
```

Comment:
I have noticed that this does not work with `== "1"` (as a string) - rather it must be `== 1` (as number) to actually show the full clipboard. Can you please double check that this is the case?

Reply:
Apologies, you're right. Re-pushed.